block/lane.go

```diff
@@ -69,3 +72,15 @@ type Lane interface {
 	// Match determines if a transaction belongs to this lane.
 	Match(ctx sdk.Context, tx sdk.Tx) bool
 }
+
+// FindLane finds a Lanes from in an array of Lanes and returns it and its index if found.
+// Returns nil, 0 and false if not found.
+func FindLane(lanes []Lane, name string) (Lane, int, bool) {
```

Comment:
Optional, but naming return types once you get to more than 2 fields in a tuple feels like a nice touch to me (but idk what the idiomatic thing to do is) 

```suggestion
func FindLane(lanes []Lane, name string) (lane Lane, index int, found bool) {
```